misc/README.md

```diff
@@ -7,6 +7,11 @@ The process for updating the `make e2e`'s input is manual.  Each time a go-algor
-| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
\ No newline at end of file
+| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
+
+## e2elive.py prerequisites
+`e2elive.py` requires access to algorand-indexer and goal binaries. If you receive an error related to the algorand-indexer binary then you need to build the indexer from scratch by running `make` in the root of the indexer repository. This will insert a binary in the `cmd/algorand-indexer` directory, where `e2elive.py` expects it by default.
+
+If the error is related to goal then the most likely cause is that the goal binary hasn't been installed as a package in the GOPATH system variable. Navigate to the go-algorand repository directory and run `make install`. If the error persists then run `export GOPATH=$(go env GOPATH)` followed by `export PATH=$PATH:$GOPATH/bin` in order to verify that the binary is accessible.</br>
```

Comment:
@algoidurovic Minor and optional - For consistency, I'd remove the closing `</br>` because all other line breaks were removed.

```suggestion
If the error is related to goal then the most likely cause is that the goal binary hasn't been installed as a package in the GOPATH system variable. Navigate to the go-algorand repository directory and run `make install`. If the error persists then run `export GOPATH=$(go env GOPATH)` followed by `export PATH=$PATH:$GOPATH/bin` in order to verify that the binary is accessible.
```